### ISSUE TITLE
Handball 16 [PCSE00681]

### ISSUE THREAD

**Issue by tasle1001** [Vita3K/compatibility#1553]
# App summary
- App name: Handball 16
- App serial: PCSE00681
- App version: 1.00

# Vita3K summary
- Version: v0.1.8
- Build number: 3193
- Commit hash: https://github.com/vita3k/vita3k/commit/9837745e
- CPU backend: Dynarmic
- GPU backend: Vulkan

# Test environment summary
- Tested by: Sam 
- OS: Windows 10/macOS/Linux Distro, Kernel Version?
- CPU: AMD/Intel?
- GPU: AMD/NVIDIA/Intel?
- RAM: 6 GB

# Issues

# Screenshots
![螢幕擷取畫面 (130)](https://github.com/Vita3K/compatibility/assets/131510427/9a03cc07-0e2d-46c0-a015-bfeffd24fc8f)
![螢幕擷取畫面 (131)](https://github.com/Vita3K/compatibility/assets/131510427/71eea5f3-86b4-45fd-962f-57abe711dde7)

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/11588367/vita3k.log)

# Recommended labels

-[crash](https://github.com/Vita3K/compatibility/labels/crash)
-[Nothing](https://github.com/Vita3K/compatibility/labels/Nothing)
- C?

**Comment by ArbestRiAgain** [comment]
Game appears to be Bootable

**Comment by othmanead** [comment]
yes it is bootable since it displays an fps counter, thank god it's not nothing

**Comment by ArbestRi02** [comment]
In latest Vita3K build, audio now plays. Pressing X button makes the screen flash yellow.
[vita3k - Copy.log](https://github.com/user-attachments/files/18231238/vita3k.-.Copy.log)
![Screenshot 2024-12-23 080325](https://github.com/user-attachments/assets/8a409f1f-46c1-424b-83e7-d68a8bd911a5)
